bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/Bookie.java

```diff
@@ -1635,4 +1635,8 @@ static LedgerDirsManager createIndexDirsManager(ServerConfiguration conf, DiskCh
             return new LedgerDirsManager(conf, idxDirs, diskChecker, statsLogger);
         }
     }
+
+    public ByteBuf getListOfEntriesOfALedger(long ledgerId) throws IOException, NoLedgerException {
+        return null;
```

Comment:
have to add corresponding method to LedgerDescriptor/LedgerDescriptorImpl and call that method here.

Reply:
fixed it